react-web/src/compositions/Timeline/Timeline.tsx

```diff
@@ -1,19 +1,26 @@
-import React from "react";
+import React, { useEffect } from "react";
```

Comment:
where `useEffect` is used?
don't you use any linter (eg elsint) that warns you about unnecessary code blocks?

Reply:
Removed,